oat-sa/extension-tao-itemqti
# feat: use config to toggle modalFeedbacks response feature

related to https://oat-sa.atlassian.net/browse/AUT-2127

### Description

Moda Feedbacks feature needs to be removed based on config provided by backend

### How to test

[Composer file](https://github.com/oat-sa/kitchen-playground/blob/tao/community-202205/files/delivery/composer.json) been used for local installation of currentGen: 

On your installation of currentGen add the following lines to the config file `config/tao/client_lib_config_registry.conf.php` : 

```
        'services/features' => array(
            'visibility' => array(
                'taoQtiItem/qtiCreator/widgets/interactions/modalFeedbacks' => 'hide'
            )
        )
```

<img width="220" alt="Screenshot 2022-04-18 at 19 18 33" src="https://user-images.githubusercontent.com/10635482/163838702-90393a0f-f629-4bf2-9364-292b7fe26875.png">


 - Enter authoring, enter authoring of some interaction, choose Response 
 - Try to update the config file like `'taoQtiItem/qtiCreator/widgets/interactions/modalFeedbacks' => 'show'` check the Modal Feedbacks panel is back
 - Try to remove the line  `'taoQtiItem/qtiCreator/widgets/interactions/modalFeedbacks' => 'show'`  from config file check the  Modal Feedbacks panel is present
